### ISSUE TITLE
[question] How to mimic a finally?

### ISSUE THREAD

**Issue by rsoesemann** [ruslan-kurchenko/sfdc-lax#4]
How can I call code at the end of the chain? No matter if an error occurs or everything went well. Something like a `finally` or a `then` after the catch.

**Comment by ruslan-kurchenko** [comment]
Hey @up2go-rsoesemann! 
That will be added to the upcoming update.
`LaxPromise` will have `finally` function to do that as in standard JavaScript Promise API

**Comment by ruslan-kurchenko** [comment]
I just created an issue for that case: https://github.com/ruslan-kurchenko/sfdc-lax/issues/5

**Comment by rsoesemann** [comment]
Can't I just add a `then()` after the `catch()`?

```
component.lax.enqueue('c.apexMethod')
    .then(
        ...
    )
    .catch(
        ...
    )
    .then(ignored => {
        component.find("spinner").hide();
    });

```
Any release dates for your update?

**Comment by ruslan-kurchenko** [comment]
@up2go-rsoesemann Yes, `catch()` function returns `LaxPromise` which is just a child of standard Promise object. That allows you to use all the benefits if Promise API. The only one thing is that `finally()` function is not adopted for Aura context yet. So you should wrap your callback with `$A.getCallback()` utility when you work with `finally()`.

Here is an example of your case:
```
component.lax.enqueue('c.getException')
    .then(result => {
           console.log('This statement will never be called', result);
    })
     .catch(errors => {
           // you can handle every action with specific logic
           component.set('v.message', errors[0].message);
     })
     .then(() => {
           return component.lax.enqueue('c.getContacts');
     }) 
     .then(contacts => {
           component.set('v.contacts', contacts);
     })
```

**Comment by ruslan-kurchenko** [comment]
@up2go-rsoesemann I'm going to release `finally()` and other features at the end of the week, or earlier.

**Comment by ruslan-kurchenko** [comment]
Released in v1.1
